pkg/hive/install.go

```diff
@@ -211,6 +213,7 @@ func (c *clusterManager) clusterDeploymentForInstall(doc *api.OpenShiftClusterDo
 				APIServerIPOverride: doc.OpenShiftCluster.Properties.NetworkProfile.APIServerPrivateEndpointIP,
 				APIURLOverride:      fmt.Sprintf("api-int.%s:6443", clusterDomain),
 			},
+			InstallAttemptsLimit: to.Int32Ptr(1),
```

Comment:
What's the motivation for this change? I'm just curious since a quick scan over the Hive code made it seem to me like this could potentially increase our install failure rate by never attempting to retry. IDK how many retry-iable  errors our Hive installs hit though.

Reply:
We have never retried failed installs in ARO, this is just changing how we ensure that only one attempt is performed to  (see removed `"hive.openshift.io/try-install-once":                "true"` annotation above)